NewKernel_d/include/CGAL/NewKernel_d/Wrapper/Cartesian_wrap.h

```diff
@@ -65,25 +65,6 @@ template<class T> struct Is_wrapper_iterator<T,true> :
 { };
 
 struct Forward_rep {
-//TODO: make a good C++0X version with perfect forwarding
-//#ifdef CGAL_CXX11
```

Comment:
Er, there is indeed a test on CGAL_CXX11 here, but it is commented out (the C++11 version doesn't quite work), and you are removing this C++11 version and the TODO. That doesn't seem right.